ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py

```diff
@@ -2647,6 +2647,46 @@ def test_paddle_instance_sgn(
     )
 
 
+# bincount
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="bincount",
+    dtypes_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        num_arrays=1,
+        min_value=1,
+        max_value=2,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+        max_dim_size=2,
```

Comment:
Why is the dimension only in `[1, 2]`? Same example as above.

Reply:
Hi, sorry was busy last week, I couldn't figure why the tests are not running so I tried duplicating the test from paddle.linalg.bincount